### PR TITLE
Changed the Documentation in README.md file.

### LINKED ISSUE #237: GCI: Setting up Donut and improving #Documentation#
**🤔 What you will need to know.**
Nothing. This issue is meant to welcome you to Open Source :) We are happy to walk you through the process.

**📋 Step by Step**

 _🙋 Claim this issue:_ Comment below. If someone else has claimed it, ask if they've opened a pull request already and if they're stuck -- maybe you can help them solve a problem or move it along!

 _📝 Mention_ your Google-Code in participant ID (userID) in the comment.

 📝Make your changes in the readme.md file 

 _💾 Commit_ your changes

 _🔀 Start a Pull Request_. There are two ways how you can start a pull request:

If you are familiar with the terminal or would like to learn it, [here is a great](https://egghead.io/courses/how-to-contribute-to-an-open-source-project-on-github) tutorial on how to send a pull request using the terminal.

You can also edit files directly in your browser and open a pull request from there.

 🏁 Done Ask in comments for a review :)
### **Description of the problem**
The documentation for the current project DONUT is vague in the readme file. The purpose of this task is to try to improve it by fill in missing parts and making it more user-friendly.

### **How to solve**
Try to install donut locally and send us a screenshot.
When you will install it from [Readme](https://github.com/codeuino/Social-Platform-Donut/tree/development)  
Try to follow the steps in the readme file. 

1) clone the repository 
2) Change Branch to development using` git checkout development `
3) Open client and server in different terminal windows.
4) Install the dependencies.
5) Run Client and server separately.

### **Expected Outcome**
Learn communication skills and documentation.

🤝 Share the issues you encounter along the way and help others solve theirs. 💬 Chime in below in a comment to cooperate with others who are also
working on this task.

✋ Report back at the above issue once you succeed, to encourage others and share what worked for you!

### **Extras**
Just follow the readme whenever you are stuck goto 
https://codeuino.slack.com/ to ask your doubts.

### **Helpful Links**
[1] Helpful Links : https://www.atlassian.com/blog/add-ons/5-real-life-examples-beautiful-technical-documentation
[2] https://gist.github.com/PurpleBooth/109311bb0361f32d87a2
[3] https://guides.github.com/features/mastering-markdown/

### PR DESCRIPTION
**Pull request by ShubhamDogra112**
Fixes #237    Changed the vague documentation in  all readme files.

### CODE REVIEW

File: server/README.md [hunk 1]
```diff
@@ -15,15 +16,22 @@ clientID:  'Github Client ID',
...
+clientID: ,
 VAPID_KEYS  :  {
 Public:  "Vapid Keys",
 Private:  "Vapid Keys"
 }}</code>
 
-3.Write <code>npm install</code> , to install node modules.
-4. Write <code>npm run dev</code>, to start server on PORT 3000.
+Add Vapid keys that are given below.
+
```
Comment (TG1999) [inline server/README.md]: VAPID Keys need to be generated everytime bythe user, so don't paste your VAPID keys here

File: README.md [hunk 2]
```diff
@@ -75,27 +75,14 @@ using the command `git clone https://github.com/username/Social-Platform-Donut.g
 
 5. Setting up the local database instance 
       
-      You need to have a MongoDB atlas account to create local
-      https://www.youtube.com/watch?v=KKyag6t98g8
-      Check out the above link to setup MongoDB atlas.
-      The mongodb URL should be pasted in credential.js files.
+      You need to install mongodb in your local machine
```
Comment (TG1999) [inline README.md]: Previous one was better

Comment (ShubhamDogra112) [reply README.md]: okk removing the vapidkeys

Comment (ShubhamDogra112) [reply README.md]: added the link for generating vapid keys

**Comment by ShubhamDogra112** [comment]
yeah you are right changed it

File: client/README.md [hunk 3]
```diff
@@ -1,8 +1,15 @@
...
+1  Open a new Terminal and switch to client directory.   (note : keep the server terminal also running.)
+2  write 'npm install' in the terminal to install the node modules.
+3 After installing write npm run serve.
+4 Open the browser and type the link 'localhost:8080'.
+5 Hurray you have done it and start contributing to it .
+
 ## Project setup
 ```
-npm install
+npm install in client directory for installing node modules
```
[the remark this answers is not in the document]
Comment (ShubhamDogra112) [reply client/README.md]: okk changed it

File: server/README.md [hunk 4]
```diff
@@ -2,8 +2,9 @@
 **Installation:**
 
  1. Make Sure you have mongodb install in your system.
- 2. Create a credential.js and add it to config folder, with structure like this.
-<code>module.exports =  {
+ 2. Create a credential.js and add it to config folder(you will find config folder in the server folder), with structure like this.
+<code>
```
[the remark this answers is not in the document]
Comment (ShubhamDogra112) [reply server/README.md]: The structure is exactly same nothing is changed just added link for vapid keys as suggested by you

**Comment by vaibhavdaren** [comment]
Learn git and redo the pull with just 1 commit.

**Comment by ShubhamDogra112** [comment]
all the commits are squashed

**Pull request merged**